src/Sdk/WebApi/WebApi/ResultsHttpClient.cs

```diff
@@ -85,6 +112,40 @@ private async Task StepSummaryUploadCompleteAsync(string planId, string jobId, s
             }
         }
 
+        private async Task StepLogUploadCompleteAsync(string planId, string jobId, Guid stepId, long lineCount, CancellationToken cancellationToken)
+        {
+            var timestamp = DateTime.UtcNow.ToString("yyyy-MM-dd'T'HH:mm:ss.fffK");
+            var request = new StepLogsMetadataCreate()
+            {
+                WorkflowJobRunBackendId= jobId,
+                WorkflowRunBackendId= planId,
+                StepBackendId = stepId.ToString(),
+                UploadedAt = timestamp,
+                LineCount = lineCount,
+            };
+
+            var stepLogsUploadCompleteRequest = new Uri(m_resultsServiceUrl, "twirp/results.services.receiver.Receiver/CreateStepLogsMetadata");
```

Comment:
Thoughts on putting `"twirp/results.services.receiver.Receiver/CreateStepLogsMetadata"`, `"twirp/results.services.receiver.Receiver/GetStepLogsSignedBlobURL"` and `"twirp/results.services.receiver.Receiver/CreateStepSummaryMetadata"` into some list of constants at the bottom of the file? Maybe even a separate file.

I imagine in the future we'll have more of these types of URLs so a single place for all of them now might make sense 🤔

Reply:
Let me sit on this for a bit.  I do want to encapsulate all the logic in a single method, so I don't have to jump around when reading code.  We should pull this out to a common place if it's used in multiple places.  Let me see if any other places would benefit from referencing those consts.